backend/src/main/resources/db/migration/V1.8__update_error_schema.sql

```diff
@@ -0,0 +1,67 @@
+update sequence_entries_preprocessed_data
+set warnings = (
+  select jsonb_agg(
+    jsonb_build_object(
+      'unprocessedFields', (
+        select jsonb_agg(
+          jsonb_build_object(
+            'name', source->>'name',
+            'type', source->>'type'
+          )
+        )
+        from jsonb_array_elements(warning->'source') as source
+      ),
+      'processedFields', (
+        select jsonb_agg(
+          jsonb_build_object(
+            'name', source->>'name',
+            'type', source->>'type'
+          )
+        )
+        from jsonb_array_elements(warning->'source') as source
+      ),
+      'message', warning->>'message'
+    )
+  )
+  from jsonb_array_elements(warnings) as warning
+)
+where warnings is not null
+  and exists (
+    select 1
+    from jsonb_array_elements(warnings) as warning
+    where warning->'source' is not null
+  );
```

Comment:
I have no problem with us having this check, but as a more academic point - I can't see any reason why we would need to have it. This migration would only be running on an empty DB except in the situation where this change is needed as far as I can see.

Reply:
(not requesting any changes!)

Reply:
Im just going to leave as is as teh current state has been tested and it doesn't break anything